Artifacts/MsSql/Structure/Admin/0131-Add-ApiClientOdsInstances-Table-Modify-GetClientForToken.sql

```diff
@@ -0,0 +1,76 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+SET QUOTED_IDENTIFIER ON;
+GO
+SET ANSI_NULLS ON;
+GO
+
+ALTER TABLE [dbo].[OdsInstances]
+    ADD [ConnectionString] NVARCHAR(MAX);
+
+IF NOT EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'dbo.ApiClientOdsInstances') AND type in (N'U'))
+BEGIN
+CREATE TABLE [dbo].[ApiClientOdsInstances](
+    [ApiClient_ApiClientId] [int] NOT NULL FOREIGN KEY REFERENCES [dbo].[ApiClients](ApiClientId),
+    [OdsInstance_OdsInstanceId] [int] NOT NULL FOREIGN KEY REFERENCES [dbo].[OdsInstances](OdsInstanceId),
+    CONSTRAINT [ApiClientOdsInstances_PK] PRIMARY KEY CLUSTERED (
+        [ApiClient_ApiClientId] ASC,
+        [OdsInstance_OdsInstanceId] ASC
+    ) WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
```

Comment:
@simpat-adam One other request here... SSMS generates a lot of extra stuff that we don't usually keep in the scripts. Specifically, we don't include things like the `WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]` and the final `ON [PRIMARY]`.

Reply:
@gmcelhanon , I agree that SSMS bloat is better off removed. I had approved based on it being consistent with the style of table creation in https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/af95cd3459f4895ac610e421987a1f685716cec2/Artifacts/MsSql/Structure/Admin/0020-Tables.sql . Are you alright with these issues you've raised regarding 131-Add-ApiClientOdsInstances-Table-Modify-GetClientForToken.sql being addressed in the PR for ODS-5790?